### DISCUSSION TITLE
increase the scroll speed

### DISCUSSION THREAD

**Discussion by Arunjenson-ss** [davidjerleke/embla-carousel#526] [Help] [answered]
how to increase the scroll speed of the carousel?
i tried using speed and scroll duration options ,but it didnt work

**Comment by davidjerleke** [comment]
@Arunjenson-SurveySparrow what Embla version are you using?

**Reply by Arunjenson-ss** [reply]
"embla-carousel-react": "^8.0.0-rc07"

**Reply by davidjerleke** [reply, marked as answer]
@Arunjenson-SurveySparrow please read [everything here](https://www.embla-carousel.com/api/options/#duration) to understand how it works.

Example:

```ts 
const options = {
  duration: 20,
}
```

Best,
David

**Reply by Arunjenson-ss** [reply]
Thanks @davidjerleke

**Reply by davidjerleke** [reply]
Did it help @Arunjenson-SurveySparrow?

**Reply by Arunjenson-ss** [reply]
yes @davidjerleke
